packages/react-native-sdk/src/components/Participant/ParticipantView/ParticipantView.tsx

```diff
@@ -174,6 +174,8 @@ const useStyles = () => {
           overflow: 'hidden',
           justifyContent: 'flex-end',
           borderRadius: theme.variants.borderRadiusSizes.md,
+          borderWidth: 2,
```

Comment:
if border width is added, no need to have a seperated style for highlightedContainer.. that must be removed